test/integration/test-submit-readable-stream.js

```diff
@@ -0,0 +1,58 @@
+var common = require('../common');
+var assert = common.assert;
+var http = require('http');
+var path = require('path');
+var mime = require('mime');
+var request = require('request');
+var fs = require('fs');
+var FormData = require(common.dir.lib + '/form_data');
+var IncomingForm = require('formidable').IncomingForm;
+var Readable = require('stream').Readable;
+
+var server = http.createServer(function(req, res) {
+
+  assert.strictEqual(req.headers['Content-Length'], undefined);
+
+  res.writeHead(200);
+  res.end('done');
+});
+
+server.listen(common.port, function() {
+
+  var form = new FormData();
+
+  var util = require('util');
+  util.inherits(CustomReadable, Readable);
+
+  function CustomReadable(opt) {
+    Readable.call(this, opt);
+    this._max = 2;
+    this._index = 1;
+  }
+
+  CustomReadable.prototype._read = function() {
+    var i = this._index++;
+    console.error('send back read data');
+    if (i > this._max)
+      this.push(null);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'this'.](https://www.codacy.com/app/form-data/form-data/file/875565524/issues/source?bid=2805153&fileBranchId=2805172#l37)